### PR TITLE
STAND-86:Consistently use CIEL (not MVP) in the UI and under the hood

### PR DESCRIPTION
**Pull request by jwnasambu**
This is what I did:
1. I renamed the MVP/MVP CIEL data file to  CIEL data file in the script. 
2. I changed MVP CIEL dictionary to CIEL dictionary so that in the process of building the standalone should not use a file called "mvp-data.sql" but rather "CIEL-dictionary.sql" and any text in the UI of the standalone (e.g. the "Starter Implementation" option) should only say CIEL, not MVP/CIEL or MVP.

Issue worked on:
https://issues.openmrs.org/browse/STAND-86

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -80,7 +80,7 @@
 				<liquibasePluginArtifactId>liquibase-maven-plugin</liquibasePluginArtifactId>
 				<liquibasePluginVersion>2.0.1</liquibasePluginVersion>
 				<liquibaseDemoDataFileName>liquibase-demo-data.xml</liquibaseDemoDataFileName>
-				<liquibaseMVPDataFileName>liquibase-mvp-data.xml</liquibaseMVPDataFileName>
+				<liquibaseCIELDataFileName>liquibase-CIEL-data.xml</liquibaseCIELDataFileName>
```
Comment (ibacher) [inline pom.xml]: I'd probably keep CIEL in lower case here, even though this means renaming the file.

File: pom.xml [hunk 2]
```diff
@@ -97,7 +97,7 @@
 				<liquibasePluginArtifactId>liquibase-plugin</liquibasePluginArtifactId>
 				<liquibasePluginVersion>1.9.5.0</liquibasePluginVersion>
 				<liquibaseDemoDataFileName>liquibase-demo-data-18x.xml</liquibaseDemoDataFileName>
-				<liquibaseMVPDataFileName>liquibase-mvp-data-18x.xml</liquibaseMVPDataFileName>
+				<liquibaseCIELDataFileName>liquibase-CIEL-data-18x.xml</liquibaseCIELDataFileName>
```
Comment (ibacher) [inline pom.xml]: Ditto

File: pom.xml [hunk 3]
```diff
@@ -223,11 +223,11 @@
 						</goals>
 					</execution>
 					<execution>
-						<id>empty-db-add-mvp-data</id>
+						<id>empty-db-add-CIEL-data</id>
```
Comment (ibacher) [inline pom.xml]: Ditto

File: pom.xml [hunk 4]
```diff
@@ -468,7 +468,7 @@
 						</configuration>
 					</execution>
 					<execution>
-						<id>zip-mvp-data</id>
+						<id>zip-CIEL-data</id>
```
Comment (ibacher) [inline pom.xml]: Ditto

File: src/main/assembly/zip-CIEL-dictionary.sql [hunk 5]
```diff
@@ -2,7 +2,7 @@
 	xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.0"
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.0 http://maven.apache.org/xsd/assembly-1.1.0.xsd">
-	<id>zip-mvp-data</id>
+	<id>zip-CIEL-data</id>
```
Comment (ibacher) [inline src/main/assembly/zip-CIEL-dictionary.sql]: Ditto

File: pom.xml [hunk 6]
```diff
@@ -477,7 +477,7 @@
 							<finalName>emptydatabase</finalName>
 							<appendAssemblyId>false</appendAssemblyId>
 							<descriptors>
-								<descriptor>src/main/assembly/zip-mvp-data.xml</descriptor>
+								<descriptor>src/main/assembly/zip-CIEL-data.xml</descriptor>
```
Comment (ibacher) [inline pom.xml]: Ditto

File: src/main/liquibase/liquibase-CIEL-data.xml [hunk 7]
```diff
@@ -5,7 +5,7 @@
     http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
 
 	<changeSet id="20120315-1000" author="standalone" dbms="mysql">
-		<comment>Preparing schema for adding mvp data</comment>
```
Comment (ibacher) [inline src/main/liquibase/liquibase-CIEL-data.xml]: Add before the `<comment>` here:

```xml
<validChecksum>3:dc136a76a36273b5cdb2f47b0eb24522</validChecksum>
```

This will prevent the change from being re-executed on upgrade

File: src/main/liquibase/liquibase-CIEL-data.xml [hunk 8]
```diff
@@ -49,7 +49,7 @@
 	</changeSet>
 
 	<changeSet id="20120315-1100" author="standalone" dbms="mysql">
-		<comment>Adding mvp data</comment>
```
Comment (ibacher) [inline src/main/liquibase/liquibase-CIEL-data.xml]: Add before the `<comment>` here:

```xml
<validChecksum>3:da27117b32a07c5b89b9df072d8d3b45</validChecksum>
```

**Comment by jwnasambu** [comment]
Thanks @ibacher am working on the proposed changes.

**Comment by jwnasambu** [comment]
@ibacher I have incorporated the proposed changes.

File: src/main/assembly/zip-CIEL-dictionary.sql (hunk 5 above)
Comment (ibacher) [reply src/main/assembly/zip-CIEL-dictionary.sql]: This should be `zip-ciel-data` and ideally the file is called `zip-ciel-data.xml` rather than `zip-CIEL-dictionary.sql`.

File: src/main/liquibase/liquibase-ciel-data-18x.xml [hunk 9]
```diff
@@ -5,7 +5,7 @@
                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
 
 	<changeSet id="20120315-1000" author="standalone" dbms="mysql">
-		<comment>Preparing schema for adding mvp data</comment>
```
Comment (ibacher) [inline src/main/liquibase/liquibase-ciel-data-18x.xml]: Please revert the changes to the data withing the file, i.e. this should still read `<comment>Preparing schema for adding mvp data</comment>`

File: src/main/liquibase/liquibase-ciel-data-18x.xml [hunk 10]
```diff
@@ -21,8 +21,8 @@
 	</changeSet>
 	
 	<changeSet id="20120315-1100" author="standalone" dbms="mysql">
-		<comment>Adding mvp data</comment>
```
Comment (ibacher) [inline src/main/liquibase/liquibase-ciel-data-18x.xml]: Ditto

File: src/main/liquibase/liquibase-ciel-data-18x.xml [hunk 11]
```diff
@@ -21,8 +21,8 @@
...
-		<sqlFile path="openmrs_concepts_${ciel.dictionary.openmrs.version}_${ciel.dictionary.version}.sql" encoding="UTF-8" />
+		<comment>Adding ciel data</comment>
+		<sqlFile path="openmrs_concepts_${ciel.dictionary.openmrs.version}_${ciel.dictionary.version}.sql" encoding="UTF-8" ></sqlFile>
```
Comment (ibacher) [inline src/main/liquibase/liquibase-ciel-data-18x.xml]: Please change this line back so it reads `<sqlFile path="openmrs_concepts_${ciel.dictionary.openmrs.version}_${ciel.dictionary.version}.sql" encoding="UTF-8" />`

File: src/main/liquibase/liquibase-ciel-data.xml [hunk 12]
```diff
@@ -5,33 +5,31 @@
...
-		<comment>Preparing schema for adding mvp data</comment>
+		<comment>Preparing schema for adding ciel data</comment>
 
 		<sql>SET FOREIGN_KEY_CHECKS=0</sql>
 		<dropTable tableName="concept" />
 		<dropTable tableName="concept_answer" />
 		<dropTable tableName="concept_class" />
 		<dropTable tableName="concept_complex" />
-		<dropTable tableName="concept_datatype" />
+		<dropTable tableName="concept_datatype" ></dropTable>
```
Comment (ibacher) [inline src/main/liquibase/liquibase-ciel-data.xml]: Please change this line back to `<dropTable tableName="concept_datatype" />`

File: src/main/liquibase/liquibase-ciel-data.xml [hunk 13]
```diff
@@ -49,8 +47,8 @@
 	</changeSet>
 
 	<changeSet id="20120315-1100" author="standalone" dbms="mysql">
-		<comment>Adding mvp data</comment>
-		<sqlFile path="openmrs_concepts_${ciel.dictionary.openmrs.version}_${ciel.dictionary.version}.sql" encoding="UTF-8" />
+		<comment>Adding ciel data</comment>
+		<sqlFile path="openmrs_concepts_${ciel.dictionary.openmrs.version}_${ciel.dictionary.version}.sql" encoding="UTF-8" ></sqlFile>
```
Comment (ibacher) [inline src/main/liquibase/liquibase-ciel-data.xml]: Please change this line back to `<sqlFile path="openmrs_concepts_${ciel.dictionary.openmrs.version}_${ciel.dictionary.version}.sql" encoding="UTF-8" />`

File: src/main/liquibase/liquibase-ciel-data.xml [hunk 14]
```diff
@@ -5,33 +5,31 @@
     http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
 
 	<changeSet id="20120315-1000" author="standalone" dbms="mysql">
-		<comment>Preparing schema for adding mvp data</comment>
```
Comment (ibacher) [inline src/main/liquibase/liquibase-ciel-data.xml]: Before this line, please add the line `<validChecksum>3:dc136a76a36273b5cdb2f47b0eb24522</validChecksum>`

File: src/main/liquibase/liquibase-ciel-data.xml [hunk 15]
```diff
@@ -49,8 +47,8 @@
 	</changeSet>
 
 	<changeSet id="20120315-1100" author="standalone" dbms="mysql">
-		<comment>Adding mvp data</comment>
```
Comment (ibacher) [inline src/main/liquibase/liquibase-ciel-data.xml]: Before this line please add the line `<validChecksum>3:da27117b32a07c5b89b9df072d8d3b45</validChecksum>`
